docs/template.rst

```diff
@@ -75,10 +71,10 @@ There are some operations you can do with a region, such as::
 
     $main_template->del('Content');
 
-    $content->set(['user' => 'Joe', 'amount' => 100]);
+    $content->set('user', 'Joe')->set('amount', 100);
     $main_template->dangerouslyAppendHtml('Content', $content->renderToHtml());
 
-    $content->set(['user' => 'Billy', 'amount' => 50]);
+    $content->set('user', 'Billy')->set('amount', 50);
```

Comment:
Would be good to have `setMulti` method for this

Reply:
we still support `set([])`, I only updated it to chained methods to make the example better